app/connectors/ExchangeObjects.scala

```diff
@@ -85,7 +85,21 @@ object ExchangeObjects {
 
   case class ReviewRequest(flag: Boolean)
 
-  case class OnlineTest(expireDate: DateTime, onlineTestLink: String, isOnlineTestEnabled: Boolean, pdfReportAvailable: Boolean) {
+  case class OnlineTest(
+    cubiksUserId: Int,
+    inviteDate: DateTime,
+    expireDate: DateTime,
+    onlineTestLink: String,
+    token: String,
+    isOnlineTestEnabled: Boolean = false,
```

Comment:
You just reformat the code but maybe "isOnlineTestEnabled" can be renamed to "isEnabled"